chia/consensus/block_body_validation.py

```diff
@@ -422,8 +428,8 @@ async def validate_block_body(
         if fees < assert_fee_sum:
             return Err.RESERVE_FEE_CONDITION_FAILED, None
 
-        # 18. Check that the assert fee amount < maximum coin amount
-        if fees > constants.MAX_COIN_AMOUNT:
+        # 18. Check that the fee amount + 0.25 < maximum coin amount. 2 is added since that's the max farmer reward
+        if fees + 0.25 > constants.MAX_COIN_AMOUNT:
```

Comment:
the comment doesn't match the code here. I think it meant to say "0.25 is added since that's the max farmer reward", right?

Reply:
0.25 -> mojos

Reply:
Fixed